### PR TITLE
Add OCI TF README for using terraform modules

### PR DESCRIPTION
**Pull request by iok0**

### CODE REVIEW

File: lw-oci-integration/tf_module/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+   }
+
+   variable "user_email" {
+   type = string
+   }
+   ```
+3. Create a file terraform.tfvars containing the following, inserting values for tenancy and email:
+    ```
+   tenancy_ocid = ""
+   user_email = ""
```
Comment (rmoles) [inline lw-oci-integration/tf_module/README.md:38]: We should probably just get users to set the var default here
```suggestion
   variable "tenancy_ocid" {
   type = string
   default = ""
   }
   variable "user_email" {
   type = string
   default = ""
   }
```

Comment (iok0) [reply lw-oci-integration/tf_module/README.md:38]: Not sure - its not really a "default"?

Comment (rmoles) [reply lw-oci-integration/tf_module/README.md:38]: IMO its slightly unusual to declare the vars in the main.tf and then set the value in the tfvars.

We should maybe even just get the variables inline. eg.
```
   module "lacework_oci_cfg_integration" {
   source = "lacework/config/oci"
   create = true
   tenancy_id = "<tenancy_ocid>"
   user_email = "<user_email>"
   }
```

File: lw-oci-integration/tf_module/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+   ```
+   terraform plan
+   ```
+3. If `Terraform Plan` runs successfully with no errors, use the following command to create the required OCI resources:
+   ```
+   terraform apply -auto-approve
+   ```
+4. List integrations to verify:
+   ```
+   lacework cloud-account list | grep -i oci
```
Comment (rmoles) [inline lw-oci-integration/tf_module/README.md:66]: nit:
```suggestion
   lacework cloud-accounts list -t OciCfg
```
```suggestion
   lacework cloud-account list | grep -i oci
```

**Review by rmoles — APPROVED** [review]

**Pull request merged**